src/test/java/org/minions/devfund/alejandro/tests/SquarelotronTest.java

```diff
@@ -0,0 +1,296 @@
+package org.minions.devfund.alejandro.tests;
```

Comment:
In fact your unit tests should be in the package 
package org.minions.devfund.alejandro

tests package is not necessary